package.json

```diff
@@ -1,27 +1,65 @@
 {
   "name": "orcajs",
-  "version": "1.1.4",
-  "main": "./dist/orca.js",
+  "version": "1.1.5",
```

Comment:
Your branch and PR is `2.0.0`

Do you want to bump the major version? If there are breaking API changes, we definitely should. If the API has changed and the only definition are typescript definitions, then a minor bump seems OK.